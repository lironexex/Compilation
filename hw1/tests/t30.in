// illegal printable char in a random place (outside of a string):

import UIKit
var num = 1.0e+6
var x : String = "some string here!"
but now we have a random illegal char ~
// we should not get here
