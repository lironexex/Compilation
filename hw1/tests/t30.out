1 COMMENT 1
3 IMPORT import
3 ID UIKit
4 VAR var
4 ID num
4 ASSIGN =
4 DEC_REAL 1.0e+6
5 VAR var
5 ID x
5 COLON :
5 TYPE String
5 ASSIGN =
5 STRING some string here!
6 ID but
6 ID now
6 ID we
6 ID have
6 ID a
6 ID random
6 ID illegal
6 ID char
Error ~
